### ISSUE TITLE
Keyboard focus for tagItem when picker is disabled

### ISSUE THREAD

**Issue by veloiya** [OfficeDev/office-ui-fabric-react#4822]
### Bug Report

- __Package version(s)__: 5.91.0

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:

When the tag picker is in disabled state the tags are not focusable by the keyboard which causes accessibility problems.

#### Actual behavior:

Tags inside disabled tags picker are not keyboard accessible.

#### Expected behavior:

Tags inside disabled tags picker should be keyboard accessible.

**Comment by betrue-final-final** [comment]
Actually they should be accessible via scan mode but not via tab/arrows. 

We have another issue pertaining to making disabled items focusable that will address this.

**Comment by veloiya** [comment]
What accessibility guidelines are we following when the tags are accessible only in scan mode?
In the case that there is a readonly tag containing a link shouldn't it be keyboard accessible.
